hrjobcontract/templates/CRM/Hrjobcontract/Import/Form/DataSource.tpl

```diff
@@ -30,10 +30,11 @@
  {* WizardHeader.tpl provides visual display of steps thru the wizard as well as title for current step *}
  {include file="CRM/common/WizardHeader.tpl"}
 
- <div id="help">
-    {ts}The API Import Wizard allows you to easily upload data against any API create method from other applications into CiviCRM.{/ts}
-    {ts}Files to be imported must be in the 'comma-separated-values' format (CSV) and must contain data needed to match the data to an existing record in your CiviCRM database.{/ts} {help id='upload'}
- </div>
+  <div class="help">
+    <p>
+      {ts}The Job Contracts Import Wizard allows you to easily upload Job Contracts from other applications into CiviHR. Contacts must already exist in your CiviHR database prior to importing Job Contracts.{/ts}
```

Comment:
That is what I thought, but with the change the text doesn't include the information stating the required file format. I think it's important to have this information here.